WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -37,20 +46,24 @@ private void CreateDynamicGrid()
 
         private void createRowContent()
         {
+            row.Clear();
+            int i = 2;
             foreach (MapObject mapObject in HospitalMap.searchResult)
             {
                 if (mapObject.MapObjectType.Equals(MapObjectType.Building))
                 {
                     createBuildingRowData(mapObject);
+                    row.Add(i, mapObject);
+                    i++;
```

Comment:
Don't repeat same lines in all if-else cases. Write them after else.

Reply:
Fixed.